04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+\chapter{Environment}
+\section{Foraging}
+
+\begin{definition} \label{def:Welfare}
+\textbf{Welfare} is the state of wealth, it can either be personal or collaborative (social).
+\end{definition}
+
+\begin{definition} \label{def:Maximising Social Welfare}
+\textbf{Maximising Social Welfare} is a strategy which attempts to maximise the total amount of welfare (resources) the islands have.
+\end{definition}
+
+
+\subsection{Foraging Background}
+
+Foraging is a fundamental requirement that is needed as it is a method of obtaining more resources from the starting amount. Foraging is based on the concept of inputting resources to get a return.
+
+\subsubsection{Underlying Concept}
+
+The primary objective of foraging in this game is to introduce a dilemma to the agents in order to investigate their behaviours and whether they work together despite different personal goals. Including two methods of foraging promotes decision making among the islands, allowing for strategic interactions to maximise personal benefits while still reaching the overarching goal of survival. 
+
+The inspiration for our concept of foraging comes from the stag hunt game. Thus, it introduces elements from it. Specifically, this approach results in a game with \textbf{no dominant strategy} and in which \textbf{an agent's choice can impact another agent's}.
```

Comment:
it may be worth including a citation/reference/definition for the stag hunt game

whichever you think works best :)